cmd/list.go

```diff
@@ -26,15 +26,80 @@ import (
 	"fmt"
 	pipeline2 "github.com/Azure/azure-pipeline-go/pipeline"
 	"strconv"
+	"strings"
 
 	"github.com/spf13/cobra"
 
 	"github.com/Azure/azure-storage-azcopy/common"
 	"github.com/Azure/azure-storage-azcopy/ste"
 )
 
+type rawListCmdArgs struct {
+	// obtained from argument
+	sourcePath string
+
+	Properties      string
+	MachineReadable bool
+	RunningTally    bool
+	MegaUnits       bool
+}
+
+var validProperties = []string{
```

Comment:
perhaps you could consider using another data structure to make the comparison easier, and also ignore the casing.

Reply:
I am using `strings.EqualFolds()` for string comparison so it is case-insensitive. As for the data-structure, I've converted it to `const` to keep the behavior similar as other similar types (`BlobType`, `BlobAccessTier`, etc.).